crt_portal/cts_forms/templates/forms/complaint_view/dashboard/filters/assignee.html

```diff
@@ -0,0 +1,14 @@
+{% extends "forms/complaint_view/dashboard/filter.html" %}
+
+{% block controls %}assigned_to{% endblock %}
+{% block label %}Intake Specialist{% endblock %}
```

Comment:
Can I content nitpick? We've been doing title-case capitalization only in the UI, so we should say "Intake specialist" not "Intake Specialist."